backend/src/server.js

```diff
@@ -29,295 +30,178 @@ const COLLECTIONS = {
 
 app.use(bodyParser.json());
 app.use(express.static(path.join(__dirname, '/../../frontend/dist')));
+app.use(passport.initialize());
```

Comment:
wait, you didn't need it previously?
then I think it should still work without it?
...I don't really know what it does in the first place, but I think you should remove it if it works fine without it.